go/plugins/vertexai/vertexai_test.go

```diff
@@ -92,7 +95,7 @@ func TestLive(t *testing.T) {
 		}
 		if final.Usage.InputTokens == 0 || final.Usage.OutputTokens == 0 || final.Usage.TotalTokens == 0 {
 			// TODO: vertexai client doesn't return stats in streaming mode.
```

Comment:
Is this still true?

Reply:
Not applicable anymore, removing the comment.